.github/workflows/deploy.yml

```diff
@@ -57,34 +59,25 @@ jobs:
       - name: Write tag to file
         run: echo "${{ github.ref_name }}" >> pems/static/version.txt
 
-      - name: Docker Login to GitHub Container Registry
-        uses: docker/login-action@v3
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
         with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+          role-to-assume: secrets.ROLE_TO_ASSUME
```

Comment:
I think this needs the `${{ }}` wrapper, no?

Reply:
Ah, yes! Thanks for catching that!